api/lds.proto

```diff
@@ -66,13 +66,27 @@ message FilterChainMatch {
 
   // If non-empty, an IP address and prefix length to match addresses when the
   // listener is bound to 0.0.0.0/::.
```

Comment:
I think we already fixed this in some other place, but this comment is misleading since this is still a useful feature even when binding to a specific address, but using USE_ORIGINAL_DST. I would clarify slightly. Same at line 73.